src/data/battle_moves.h

```diff
@@ -224,8 +221,10 @@ const struct BattleMove gBattleMoves[MOVES_COUNT_Z] =
         .split = SPLIT_SPECIAL,
         .zMoveEffect = Z_EFFECT_NONE,
         .highCritRatio = TRUE,
+        .twoTurnMove = TRUE,
         .sleepTalkBanned = TRUE,
         .instructBanned = TRUE,
+        //.windMove = TRUE,
```

Comment:
As discussed on the Discord, let's add `B_EXTRAPOLATED_MOVE_FLAGS` (on by default) to handle this and Assist.

Reply:
Done, added the flag and applied it.